docker/README.md

```diff
@@ -73,32 +78,72 @@ In addition, the following legacy tags are available:
 
 ## Build a [Devito] image
 
-To build the images yourself, all you need is to run the standard build command using the provided Dockerfile. The main difference between the CPU and GPU images will be the base image that will be used.
+To build the images yourself, you only need to run the standard build command using the provided Dockerfile. The main difference between the CPU and GPU images will be the base image used.
 
 
-To build the (default) CPU image, simply run:
+To build the (default) CPU image, run:
 
 ```bash
 docker build --network=host --file docker/Dockerfile.devito --tag devito .
 ```
 
-And to build the GPU image with `openacc` offloading and the `nvc` compiler, simply run:
+To build the GPU image with `openacc` offloading and the `nvc` compiler, run:
 
 ```bash
-docker build --build-arg base=devitocodes/base:nvidia-nvc --network=host --file docker/Dockerfile.devito --tag devito .
+docker build --build-arg base=devitocodes/bases:nvidia-nvc --network=host --file docker/Dockerfile.devito --tag devito .
 ```
 
 or if you wish to use the `clang` compiler with `openmp` offloading:
 
 ```bash
-docker build --build-arg base=devitocodes/base:nvidia-clang --network=host --file docker/Dockerfile --tag devito .
+docker build --build-arg base=devitocodes/bases:nvidia-clang --network=host --file docker/Dockerfile --tag devito .
+```
+
+and finally, for AMD architectures:
+
+```bash
+docker build --build-arg base=devitocodes/bases:amd --network=host --file docker/Dockerfile --tag devito .
 ```
 
-and finally for AMD architectures:
+
+## Debugging a base image
+
+This section is primarily helpful in contributing to or debugging the Docker images.
+
+To build the base image yourself locally, you need to run the standard build command using the provided Dockerfile.
 
 ```bash
-docker build --build-arg base=devitocodes/base:amd --network=host --file docker/Dockerfile --tag devito .
+# To locally build the base CPU image for gcc, run: 
+docker build . --file docker/Dockerfile.cpu --tag devito-gcc --build-arg arch=gcc
+# To locally build the Devito image using the previously built base, run: 
```

Comment:
Maybe these should _not_ be comments 

```
# To locally ...
```

Just like in the sections above they could be normal text (nitpicking, but for homogeneity...)